net/openssh/Portfile

```diff
@@ -5,8 +5,9 @@ PortSystem          1.0
 PortGroup           compiler_blacklist_versions 1.0
 
 name                openssh
-version             9.9p2
-revision            1
+version             10.0p2
+distname            openssh-10.0p1
```

Comment:
CI fails because the `ssh-copy-id` subport re-sets `$distname` to `openssh-10.0p2` in line 264, and that file isn't on the CDN.